src/modules/auth/header.js

```diff
@@ -57,9 +58,9 @@ function HeaderComponent(props) {
 			</DropdownToggle>
 			<DropdownMenu>
 				<DropdownItem header tag="div" className="text-center"><strong>{t('AuthHeaderDropdown|My account')}</strong></DropdownItem>
-				<DropdownItem tag="a" href={access_control_url}>
+				{active_tenant && <DropdownItem tag="a" href={access_control_url}>
```

Comment:
You don't need to have a tenant to access "Access control", the access control works even without tenant.
It is optional.

Reply:
Refactored in those commits: https://github.com/TeskaLabs/asab-webui/pull/105/commits/f7ef44d91772ab77042043605d97dfa98b4999f5 & https://github.com/TeskaLabs/asab-webui/pull/105/commits/6253f673b1f1ad3b39b01ef5fe0b0722c1db2920